core/src/wallet/cosmos/explorers/GaiaCosmosLikeBlockchainExplorer.cpp

```diff
@@ -387,15 +387,15 @@ namespace ledger {
 
             return _http->POST("/txs", transaction, headers)
                         .json().template map<String>(_executionContext, [] (const HttpRequest::JsonResult& result) -> String {
-                            // TODO : the content of this payload actually depends on the mode of the transaction
+                            // The content of this payload actually depends on the mode of the transaction
```

Comment:
When a TODO becomes a feature! :D

Reply:
Yes, I was thinking that we should handle the payload of the request depending on the mode that clients call the functions with.

And then I realized that I should never try to guess what the end-user wants out of the payload, so give them everything and let them handle that :D

Payload goes from almost nothing (in fire and forget mode) to 
```
{
  "check_tx": {
    "code": 0,
    "data": "data",
    "log": "log",
    "gas_used": 5000,
    "gas_wanted": 10000,
    "info": "info",
    "tags": [
      "",
      ""
    ]
  },
  "deliver_tx": {
    "code": 5,
    "data": "data",
    "log": "log",
    "gas_used": 5000,
    "gas_wanted": 10000,
    "info": "info",
    "tags": [
      "",
      ""
    ]
  },
  "hash": "EE5F3404034C524501629B56E0DDC38FAD651F04",
  "height": 0
}
```